### DISCUSSION TITLE
":guilabel:" is displayed in Weblate Documentation. (2)

### DISCUSSION THREAD

**Discussion by baffclan** [WeblateOrg/weblate#5857] [General] [answered]
I found three ":guilabel:" in the glossary of documentation.
https://docs.weblate.org/en/latest/user/glossary.html

A space is required after the "↓".

1. https://docs.weblate.org/en/latest/user/glossary.html#not-translatable-terms
 -> https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=b6256b7c8a343de6

2. https://docs.weblate.org/en/latest/user/glossary.html#forbidden-translations
 -> https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=a8c2aa3653f617e6

3. https://docs.weblate.org/en/latest/user/glossary.html#terminology
 -> https://hosted.weblate.org/translate/weblate/documentation/en/?checksum=6f730c9b1bcad129

**Comment by nijel** [comment, marked as answer]
Thanks for pointing out, fixed in 0fa425e2f94aec7aebfdc4234e01d584309a8607.

**Reply by baffclan** [reply]
Thanks for fixing this!

Er, was the typo problem good here?
Or is it a issue or a term comment with mention?

**Reply by nijel** [reply]
Space was missing in the beginning of the tag
